qiskit/circuit/library/n_local/n_local.py

```diff
@@ -66,6 +66,94 @@ class NLocal(BlueprintCircuit):
 
     If specified, barriers can be inserted in between every block.
     If an initial state object is provided, it is added in front of the NLocal.
+
+    Examples:
+        >>> q0,q1,q2,q3 = 0,1,2,3
+
+        >>> num_qubits = 3
+        >>> rotation_blocks = 'ry'      # this could be a circuit, list of circuit, circuit instruction ('ry'), list of circuit instructions, or None for default values
+        >>> entanglement_blocks = 'cx'  # this could be a circuit, list of circuit, circuit instruction ('ry'), list of circuit instructions, or None for default values
```

Comment:
I suspect these two lines are too long. In addition, the comment in the second one should be with cx.